TestProject/Cells/ImageCollectionViewCell.swift

```diff
@@ -18,16 +20,7 @@ class ImageCollectionViewCell: UICollectionViewCell {
 	
 	var imageURL: URL! {
 		didSet {
-			downloadImage()
-		}
-	}
-	
-	private func downloadImage() {
-		if let data = try? Data(contentsOf: self.imageURL) {
-			if let image = UIImage(data: data) {
-				self.image.image = image
-				self.delegate?.imageDownloaded()
-			}
+            image.kf.setImage(with: self.imageURL)
```

Comment:
Ad SDK we try to avoid additional third party tools. Could you please fix without any dependencies?

Reply:
Also memory still grows, as I see.